src/transformers/modeling_tf_utils.py

```diff
@@ -106,6 +106,15 @@ def get_config(self):
     return cls
 
 
+class TFCausalLanguageModelingLoss:
+    def compute_loss(self, labels, logits):
+        loss_fn = tf.keras.losses.CategoricalCrossentropy(
+            from_logits=True, reduction=tf.keras.losses.Reduction.NONE
```

Comment:
I set the reduction to `Reduction.NONE` as was done for the other losses -> is that correct @LysandreJik @jplu ?

Reply:
Yes, beause NONE is for making the loss computation compliant with training across multiple devices. And the reduction is let to the trainer.